app/views/mentorships/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="container mentorships-index">
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Mentor</th><th>Mentee</th><th>Question</th><th>Created</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @mentorships.each do |mentorship| %>
+        <tr class="mentorship--row">
+          <td class="mentorship--mentor">
+            <% if mentorship.mentor == current_user %>
+              <%= current_user.full_name %>
+            <% elsif mentorship.mentor %>
+              <%= link_to mentorship.mentor.full_name, user_path(mentorship.mentor) %>
+            <% else %>
+              <i>Pending</i>
+              <%= link_to "Cancel", mentorship_path(mentorship), method: :delete %></br>
+            <% end %>
+          </td>
+          <td class="mentorship--mentee">
+            <% if mentorship.mentee == current_user %>
+              <%= current_user.full_name %>
+            <% else %>
+              <%= link_to mentorship.mentee.full_name, user_path(mentorship.mentee) %>
```

Comment:
Did we confirm if the current_user will have access to this user's page?

Reply:
Oh; I didn't clarify that with Kate, but I should add that; thanks. Good Catch

Reply:
This has already been clarified with Kate in the past, we want them to be able to see their mentors/mentees. I just don't remember how we wrote the code.

Reply:
I could use 5 minutes of help with this